Library/Homebrew/formula_creator.rb

```diff
@@ -138,6 +140,9 @@ def install
                                   "--disable-dependency-tracking",
                                   "--disable-silent-rules",
                                   "--prefix=\#{prefix}"
+        <% elsif mode == :crystal %>
+            system "shards", "build", "--release"
+            bin.install "bin/#{name}"
```

Comment:
```suggestion
            bin.install bin/name
```

Reply:
Yeah, it makes sense. I guess there is million ways to do things in Ruby and I have tendency to constantly pick the wrong one 🤣

Reply:
@dawidd6 Wait a minute, this references bin dir inside prefix while compilation outputs are stored inside bin dir under build path? Am I missing something? 🤔

Reply:
Oh man, sorry. I'm wrong on this one, you are right!

Reply:
No worries 👍